users/models.py

```diff
@@ -46,9 +46,31 @@ def create_superuser(self: "UserManager", email: str, nickname: str, password: s
         return self._create_user(email, nickname, password, **extra_fields)
 
 
-class User(AbstractUser):
-    nickname_validator = UnicodeUsernameValidator()
+class UserSettings(models.Model):
+
+    # https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes
+    content_lang = models.CharField(
+        max_length= 2,
+        choices= [
+            ("en", "English")
+        ]
+    )
+   #  https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+    location = models.CharField(
+        max_length=2,
+        choices = [
+            ("US", "United States")
+        ]
+    )
+    
+    is_beta = models.BooleanField(default=False)
```

Comment:
What is this is beta?

Reply:
In reddit it is flag deciding if user wants to user beta version  of reddit. I can remove it if needed.